### ISSUE TITLE
Keyboard navigation doesn't work on chord symbols attached to fretboard diagrams

### ISSUE THREAD

**Issue by avvvvve** [musescore/MuseScore#25983]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

There are two symptoms of this issues that I noticed:

1. (Start of video below)
When editing regular chord symbols (without fretboard diagrams), you can press space or semicolon to move the cursor to the next beat ([among other actions listed here for moving the cursor different distances](https://musescore.org/en/handbook/4/chord-symbols#commands)). None of these actions work when editing chord symbols attached to fretboard diagrams.

2. (0:13 in video below)
When using <kbd>Alt</kbd> + <kbd>Left/right arrow keys</kbd> to select the previous/next element in the score, chord symbols attached to fretboard diagrams are skipped.

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/a8504124-9f09-431d-8826-baec479a264a

### What is the latest version of MuseScore Studio where this issue is present?

Nightly

### Regression

No.

### Operating system

macOS 15.1

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
Another issue:

If you are using space to navigate through chord symbols that don't have fretboard diagrams, then hit a chord symbol that _does_ have a diagram, the chord symbol will enter edit mode but the existing text will be deleted. Then, you can't undo to get the chord symbol back.

https://github.com/user-attachments/assets/819b4424-491b-4b41-9d24-db7d0900b857

**Comment by DmitryArefiev** [comment]
Fixed in #28327
